governance/votes/Executive vote - December 10, 2021.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Template - [Executive Vote] Parameter Changes, Switching MKR Vesting Source - December 10, 2021
+summary: Changes to wstETH-A and MATIC-A Parameters, replacing current MKR vesting streams with streams funded by the Protocol Treasury MKR.
+date: 2021-12-10T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] Parameter Changes, Switching MKR Vesting Source - December 10, 2021
+
+The Governance Facilitator(s) and the Protocol Engineering Core Unit have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://community-development.makerdao.com/en/learn/governance/how-voting-works/) to learn how voting works, and this [wallet setup guide](https://community-development.makerdao.com/en/learn/governance/voting-setup/) to set up your wallet to vote.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+- wstETH-A changes to [Debt Ceiling Instant Access Module](https://makerdao.world/en/learn/governance/module-dciam) and Liquidation Parameters, as detailed below. 
+- MATIC-A [Debt Ceiling Instant Access Module](https://makerdao.world/en/learn/governance/module-dciam) changes, as detailed below.
+- Previously approved MKR Vesting Streams will be replaced with MKR sourced from the Protocol Treasury (Pause Proxy) instead of minting new MKR.
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes listed above are subject to the [GSM Pause Delay](https://community-development.makerdao.com/en/learn/governance/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### wstETH-A Parameter Changes
+
+As per this successful [governance poll](https://vote.makerdao.com/polling/QmYuK441?network=mainnet#poll-detail), the following wstETH-A parameters will be updated if this executive proposal passes.
+
+* Increase the wstETH-A [Maximum Debt Ceiling (`line`)](https://makerdao.world/en/learn/governance/module-dciam) from 50 million DAI to 200 million DAI. 
+* Increase the wstETH-A [Target Available Debt (`gap`)](https://makerdao.world/en/learn/governance/module-dciam) from 3 million DAI to 20 million DAI.
+* Decrease the wstETH-A [Debt Ceiling Increase Cooldown (`ttl`)](https://makerdao.world/en/learn/governance/module-dciam) from 8 hours to 6 hours.
```

Comment:
(I'm pretty sure this is a deliberate choice for clarity, but anyway: seems like this parameter is actually called "Ceiling Increase Cooldown", not "Debt...").